spinalcordtoolbox/scripts/sct_compute_mscc.py

```diff
@@ -67,28 +125,371 @@ def get_parser():
     return parser
 
 
-def mscc(di, da, db):
-    return (1 - float(di) / ((da + db) / float(2))) * 100
```

Comment:
I notice that this PR is completely removing the older, simpler `sct_compute_mscc` functionality.

In that case, we may want to keep in mind the following issue: 

- https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3201